src/node-zk.cpp

```diff
@@ -637,17 +642,18 @@ class ZooKeeper: public Nan::ObjectWrap {
     }
 
     static void ACreate(const Nan::FunctionCallbackInfo<Value>& info) {
-        A_METHOD_PROLOG(4);
+        A_METHOD_PROLOG(5);
 
         Nan::Utf8String _path (toString(info[0]));
         uint32_t flags = toUint(info[2]);
 
+        int32_t ttl = toInt(info[3]);
```

Comment:
Notes (to test): What happens when null or undefined is passed?

Reply:
Aha, now I found the `const realTtl = ttl || -1;` in the javascript code, that will prevent null, 0 or undefined to be passed in.